<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="Nilesh" failures="1" tests="2" name="Default test(failed)" time="40.571" errors="0" timestamp="2022-10-20T16:58:00 IST">
  <testcase classname="TestClass.DemoFail" name="smalllogoTest" time="0.033">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestClass.DemoFail.smalllogoTest(DemoFail.java:55)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- smalllogoTest -->
  <testcase classname="TestClass.DemoFail" name="finbtnTest" time="0.056"/>
  <testcase classname="TestClass.DemoFail" name="checklabelTest" time="0.0">
    <ignored/>
  </testcase> <!-- checklabelTest -->
  <testcase classname="TestClass.DemoFail" name="biglogoTest" time="0.0">
    <ignored/>
  </testcase> <!-- biglogoTest -->
  <testcase classname="TestClass.DemoFail" name="currenturlTest" time="0.0">
    <ignored/>
  </testcase> <!-- currenturlTest -->
  <testcase classname="TestClass.DemoFail" name="textlabelTest" time="0.0">
    <ignored/>
  </testcase> <!-- textlabelTest -->
  <testcase classname="TestClass.DemoFail" name="headerTest" time="0.0">
    <ignored/>
  </testcase> <!-- headerTest -->
</testsuite> <!-- Default test(failed) -->
